com.unity.cinemachine/Runtime/Behaviours/Cinemachine3rdPersonAim.cs

```diff
@@ -79,7 +79,7 @@ void DrawReticle(CinemachineBrain brain)
         {
             if (!brain.IsLive(VirtualCamera) || brain.OutputCamera == null)
                 CinemachineCore.CameraUpdatedEvent.RemoveListener(DrawReticle);
-            else if (AimTargetReticle != null)
+            if (AimTargetReticle != null && brain.OutputCamera != null && brain.IsLive(VirtualCamera))
```

Comment:
Why is this file being changed at all?
This is not an improvement over the original code.  You are repeating tests, where an else was sufficient.